face-detection/demos/shared/util.js

```diff
@@ -38,7 +38,14 @@ export function isMobile() {
 async function resetBackend(backendName) {
   const ENGINE = tf.engine();
   if (!(backendName in ENGINE.registryFactory)) {
-    throw new Error(`${backendName} backend is not registed.`);
+    if(backendName === 'webgpu') {
+      alert('webgpu backend is not registered. Your browser may not support WebGPU yet. To test this backend, please use a supported browser, e.g. Chrome canary with --enable-unsafe-webgpu flag');
+      STATE.backend = !!STATE.lastTFJSBackend ? STATE.lastTFJSBackend : 'tfjs-webgl';
+      showBackendConfigs();
+      return;
+    } else {
+      throw new Error(`${backendName} backend is not registered.`);
```

Comment:
Nit: If we don't rely on this function throwing an error anywhere, we can just show a warning and revert to a working backend, even if it's not webgpu.
```suggestion
    } else {
      STATE.backend = !!STATE.lastTFJSBackend ? STATE.lastTFJSBackend : 'tfjs-webgl';
      showBackendConfigs();
      console.warn(`${backendName} backend is not registered. Reverted to ${STATE.backend}.`);
```